### ISSUE TITLE
Feature request: randomize lexemes instead of forms

### ISSUE THREAD

**Issue by Ainali** [egils-consulting/LexUse#7]
When the tool is adding just one example on a lexeme and then moves on, the feeling of just poking around instead of doing useful work is huge. I would rather have it go through all the forms on one lexeme before moving on to the next one. This will also have the added benefit that when I encounter a lexeme that has been edited by this tool I can be sure that there are no easy hits in the databases it uses.

**Comment by dpriskorn** [comment]
I agree that would be an improvement. Right now we only get forms and then randomly choose from that. It would require another query.
Pseudo code could be: find lexemes in Swedish which has forms, but some of the forms do not currently have examples.
Then loop through them
Get all forms of the current one without an example
loop through and search examples

**Comment by Ainali** [comment]
Here is a starting point: https://w.wiki/zJi
It gets lexemes in Swedish that have forms and senses but no examples at all. It can be improved on by also including those where some forms are still without examples but this already have >2800 hits, so we can get a lot done just with this.
